src/test/resources/debug-config/formats.txt

```diff
@@ -0,0 +1,13 @@
+application/postscript
+application/rtf
+application/pdf
+application/msword
+text/plain
+text/html
+text/sgml
+text/xml
+image/jpeg
+image/gif
+image/tiff
+video/quicktime
+video/mpeg1
```

Comment:
Seems like a good idea, but the problem is that I already rely on this file in my tests, so I guess we better keep this copy around